Spring_Session7/Spring-Session1-Collection/src/com/metacube/TextEditor.java

```diff
@@ -0,0 +1,46 @@
+package com.metacube;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * class containing methods for texteditor functionality
+ */
+public class TextEditor {
+	
+	private List<SpellChecker> spellCheckerList = null;
+	private Set<SpellChecker> spellCheckerSet = null;

```

Comment:
no need to define with null. 
If we do not instantiate a variable it is already null.